www/main/settings/components/SettingViewer.jsx

```diff
@@ -62,15 +45,19 @@ const SettingViewer = () => {
     fontSize: `${gurbaniFontSize * 3}px`,
   };
 
-  const translationStyles = {
+  const content1Styles = {
+    display: content1Visibility ? 'block' : 'none',
     fontSize: `${content1FontSize * 3}px`,
+    fontWeight: 'normal',
   };
 
-  const transliterationStyles = {
+  const content2Styles = {
+    display: content2Visibility ? 'block' : 'none',
     fontSize: `${content2FontSize * 3}px`,
   };
 
-  const teekaStyles = {
+  const content3Styles = {
+    display: content3Visibility ? 'block' : 'none',
     fontSize: `${content3FontSize * 3}px`,
```

Comment:
Veerji, this piece of code is repetitive. In case of change is required in the future there will be multiple places to change, and to follow the DRY principle, we can refactor this to something like:

```js
const contents = [
  { visibility: content1Visibility, fontSize: content1FontSize },
  { visibility: content2Visibility, fontSize: content2FontSize },
  { visibility: content3Visibility, fontSize: content3FontSize }
];

const contentStyles = contents.map((content, index) => ({
  display: content.visibility ? 'block' : 'none',
  fontSize: `${content.fontSize * 3}px`,
}));

```

Reply:
Looks good, although a little worried about code readability? Because we use these styles at different places, and it would have to be applied like contentStyles[0], contentStyles[1]. 🤔

Reply:
Okay, if you are worried about the index count we can refactor it using reduce method.

```js
const contents = [
  { visibility: content1Visibility, fontSize: content1FontSize },
  { visibility: content2Visibility, fontSize: content2FontSize },
  { visibility: content3Visibility, fontSize: content3FontSize }
];

const contentStyles = contents.reduce((acc, content, index) => {
  acc[`content${index + 1}`] = {
    display: content.visibility ? 'block' : 'none',
    fontSize: `${content.fontSize * 3}px`,
  };
  return acc;
}, {});

console.log(contentStyles.content1) // Accessible like this way as well
```

If you have a better approach in your mind to follow the DRY principle we can talk about that as well veerji.

Reply:
How about this, used a function that returns the style. Initially I thought of contentAttributes as an array instead of object, but then didn't want to abstract it with adding one to index value to get id. In future, if we have another content Id, we can continue with this same logic.

```javascript
  const contentAttributes = {
    1: { visibility: content1Visibility, fontSize: content1FontSize },
    2: { visibility: content2Visibility, fontSize: content2FontSize },
    3: { visibility: content3Visibility, fontSize: content3FontSize },
  };

  const getContentStyles = (contentId) => ({
    display: contentAttributes[contentId].visibility ? 'block' : 'none',
    fontSize: `${contentAttributes[contentId].fontSize * 3}px`,
    fontWeight: 'normal',
  });
```